### ISSUE TITLE
$SP().people get first OU/directory in AD for an user

### ISSUE THREAD

**Issue by Pepozzo** [Aymkdn/SharepointPlus#121]
Hello!

I'm trying to get informations about an user. $SP().people is working fine and it retrieves the parent OU for a specific user. My AD tree is organized like this: COUNTRY->ACCOUNTS->IT/ACCOUNTING/GENERAL MANAGER etc etc...where Country can be "United States", "Canada", etc...
I would like to retrieve this first directory from an user.

Is that possible with SharePointPlus?  

Thanks

**Comment by Aymkdn** [comment]
`people` uses the username so I'm not sure to understand your question?! Do you mean `$SP().addressbook`?

**Comment by Pepozzo** [comment]
I've tried $SP().addressbook also, but with same results.

I mean, I'm getting these infos:
`CN=FirstName Last Name,OU=IT,OU=Accounts,DC=DomainName,DC=local`

My user is in the IT organization unit, so informations are correct.

But my AD is organized like this:
```
DOMAIN: DomainName
-- OU: Country1
---- OU: Accounts
------ OU: IT
-- OU: Country2
---- OU: Accounts
------ OU: IT
```
So I would like to retrieve the first OU of the tree for a specific user, if it is possible.

**Comment by Aymkdn** [comment]
This function returns the information for a user, like "europe\johndoe", meaning the user "johndoe" in domain "europe". There is no special option and it's based on a Sharepoint web service (check the code to find which one... I'm on vacation with only my phone and it crashes Chrome when I try to read the source code). By finding the web service you can check if there is more parameters available but I don't think so.

Long story short : it's not possible to do it with SharepointPlus.

**Comment by Pepozzo** [comment]
Thank you! I don't want to bother you on vacation, so I'll continue to search a solution.
Just to clarify my scenario: my OUs for different countries are under the same domain. And I'm searching user with the right syntax as documented (i:0#.w|domain\\login). So everything is running correctly, just is not returning first level of OUs.
Maybe it is just an issue with SharePoint Web Services (that I think it refers to the User profile services and it is not configured in the right way).
Thank you again, enjoy your days off.
